x-pack/test/api_integration/apis/uptime/rest/certs.ts

```diff
@@ -78,7 +78,7 @@ export default function ({ getService }: FtrProviderContext) {
         );
       });
       after('unload test docs', () => {
```

Comment:
Needs async for this await
```suggestion
      after('unload test docs', async () => {
```